src/args.js

```diff
@@ -54,4 +54,26 @@ module.exports = require('yargs')
     default: 1,
     describe: 'Set the speed of test execution (0.01 ... 1)',
     type: 'number'
+  })
+  .option('concurrency', {
+    alias: 'c',
+    default: 1,
+    describe: 'Specifies that tests should run concurrently',
+    type: 'number'
+  })
+  .option('app', {
+    alias: 'a',
+    default: null,
+    describe: 'Executes the specified shell command before running tests. Use it to launch or deploy the application you are going to test.',
+  })
+  .option('appInitDelay', {
+    default: 1000,
```

Comment:
I think the default should be `0` or `undefined` since it is optional in the original API.

Reply:
The value 1000 was used here https://github.com/DevExpress/testcafe/blob/master/src/runner/bootstrapper.js but I think 0 is fine as well.